pyproject.toml

```diff
@@ -18,7 +18,19 @@ exclude = [
 target-version = "py310"
 
 [tool.ruff.lint]
-extend-select = ["I"]
+select = [
+    # pycodestyle
+    "E",
+    "W",
+    # Pyflakes
+    "F",
+    # isort
+    "I",
+    # pyupgrade
+    "UP",
```

Comment:
Shouldn't this PR only include this bit?

Reply:
Resolved in [bfa9b02](https://github.com/PrairieLearn/PrairieLearn/pull/11059/commits/bfa9b02fd891eb5ba7ea80f25730a10aef9b480b)